steps/train.py

```diff
@@ -16,6 +16,5 @@ def estimator_fn(estimator_params: Dict[str, Any] = {}):
     #
     # FIXME::OPTIONAL: return a scikit-learn-compatible regression estimator with fine-tuned
     #                  hyperparameters.
-    from sklearn.linear_model import SGDRegressor
 
-    return SGDRegressor(**estimator_params)
+    raise NotImplementedError
```

Comment:
We might need to fix this test: https://github.com/mlflow/recipes-regression-template/blob/main/tests/train_test.py